src/workflows/bidsChangeSuffix.m

```diff
@@ -0,0 +1,139 @@
+function bidsChangeSuffix(varargin)
```

Comment:
Not instead of.

It can be run on ANY BIDS dataset any time. 

So you could even use to fix a typo in your file suffix after converting your DICOM.